app/models/opencollective/FallbackMembers.scala

```diff
@@ -0,0 +1,66 @@
+package models.opencollective
+
+/**
+ * Pull from OpenCollective on 11/23/2021 so that we have something to fall back to when OpenCollective is either down, or if
```

Comment:
What if we just pulled this directly from OpenCollective on startup, rather than having the fallback? It'll delay startup a bit but that's probably ok.

Reply:
> What if we just pulled this directly from OpenCollective on startup, rather than having the fallback? It'll delay startup a bit but that's probably ok.

The data do get pulled on startup. The fallback is intended as last resort in case there is a connection problem to the OpenCollective api. In normal circumstances the fallback should never be seen. I just ran the app in prod mode and here is the proof:
```
[info] s.o.CachingMembersSummariser - Loaded 30 members for OpenCollective
[info] p.c.s.AkkaHttpServer - Listening for HTTP on /127.0.0.1:9000
```

> Is there any plan to update the fallback periodically? Or, just leave it as is with the intent that the fallback is (hopefully) never seen?

The fallback itself is hardcoded, however the data will be synced every 24 hours anyway. However, it is also very unlikely the fallback will ever be seen, see explanation above. I thought its better to have a hardcoded fallback with a couple of people/faces than showing nothing.

> For the real website we can cache it if rate-limit is a problem.

The data will fetched/synced once every 24 hours, rate-limiting will not be a problem. Data will be cached within this 24 hours, please see `CachingMembersSummariser`

Probably the worst thing that can happen right now is, that the app starts up, there is a connection problem to the oc api, and then the fallback members will be shown for 24 hours before they get synced (and hopefully the api is up again).

Reply:
Thanks for the feedback everyone. If you don't like the fallback, I can just remove it.
@BillyAutrey Thanks for proofreading, commited the changes.